src/ngraph_encapsulate_op.cc

```diff
@@ -262,9 +306,16 @@ class NGraphEncapsulateOp : public tf::OpKernel {
       m_ng_functions;
   std::map<std::shared_ptr<ngraph::Function>, std::vector<const void*>>
       m_last_used_src_ptrs_map;
+  std::unordered_map<std::shared_ptr<ngraph::Function>,
+                     std::vector<ng::runtime::TensorView>>
+      m_ng_function_to_inputs;
```

Comment:
(Didn't catch this til I got down here.) What are `m_ng_function_to_inputs` and `_outputs` for?